ci/blackbox/elasticachebroker_test.go

```diff
@@ -157,29 +155,32 @@ var _ = Describe("ElastiCache Broker Daemon", func() {
 				Expect(found).To(Equal(2))
 			})
 
-			By("updating parameters", func() {
-				updateParams := fmt.Sprintf(`{"maxmemory_policy": "noeviction"}`)
+			By("updating the maxmemory_policy", func() {
+				updateParams := `{"maxmemory_policy": "noeviction"}`
 				oldPlanID := planID
 				oldServiceID := serviceID
-				code, _, err := brokerAPIClient.UpdateInstance(instanceID, serviceID, planID, oldPlanID, oldServiceID, brokerAPIClient.DefaultOrganizationID, brokerAPIClient.DefaultSpaceID, updateParams)
+				code, _, _, err := brokerAPIClient.UpdateInstance(instanceID, serviceID, planID, oldPlanID, oldServiceID, brokerAPIClient.DefaultOrganizationID, brokerAPIClient.DefaultSpaceID, updateParams)
 				Expect(err).ToNot(HaveOccurred())
 				Expect(code).To(Equal(202))
+				newServiceParams, err := brokerAPIClient.GetServiceParams(instanceID)
+				Expect(err).ToNot(HaveOccurred())
+				expectedCacheParameter := providers.CacheParameter{
+					ParameterName:  "maxmemory-policy",
+					ParameterValue: "noeviction",
+				}
+				Expect(newServiceParams.CacheParameters).To(ContainElement(expectedCacheParameter))
 			})
 
-			By("checking that the cache parameter group has been updated", func() {
-				replicationGroupID := redis.GenerateReplicationGroupName(instanceID)
-				res, err := elasticacheService.DescribeCacheParameters(&elasticache.DescribeCacheParametersInput{
-					CacheParameterGroupName: aws.String(replicationGroupID),
-				})
+			By("updating the preferred_maintenance_window", func() {
+				updateParams := `{"preferred_maintenance_window": "tue:17:51-tue:19:45"}`
+				oldPlanID := planID
+				oldServiceID := serviceID
+				code, _, _, err := brokerAPIClient.UpdateInstance(instanceID, serviceID, planID, oldPlanID, oldServiceID, brokerAPIClient.DefaultOrganizationID, brokerAPIClient.DefaultSpaceID, updateParams)
 				Expect(err).ToNot(HaveOccurred())
-				found := 0
-				for _, p := range res.Parameters {
-					if *p.ParameterName == "maxmemory-policy" {
-						found++
-						Expect(*p.ParameterValue).To(Equal("noeviction"))
-					}
-				}
-				Expect(found).To(Equal(1))
+				Expect(code).To(Equal(202))
+				newServiceParams, err := brokerAPIClient.GetServiceParams(instanceID)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(newServiceParams.PreferredMaintenanceWindow).To(Equal("tue:17:51-tue:19:45"))
```

Comment:
Do we ever test _provisioning_ with parameters?

Reply:
we do now